LLama/LLamaSharp.csproj

```diff
@@ -28,6 +28,7 @@
     <Platforms>AnyCPU;x64;Arm64</Platforms>
     <PackageId>LLamaSharp</PackageId>
     <Configurations>Debug;Release;GPU</Configurations>
+    <GenerateAssemblyInfo>false</GenerateAssemblyInfo>
```

Comment:
What does this do?

Reply:
I ran into issues during build, where AssemblyInfo attributes were flagged as duplicated. See: https://stackoverflow.com/questions/10311347/duplicate-assemblyversion-attribute

Reply:
<img width="905" alt="Screenshot 2023-12-07 at 8 47 17 AM" src="https://github.com/SciSharp/LLamaSharp/assets/3707196/1cda3167-4fc8-4470-81b1-9c8ecde85b3b">

Reply:
I tried to remove the attributes in the csproj files other than the `LLamaSharp.csproj` but that didn't have any effect. Then I tried the attribute to stop generating them all-together and it worked again for me.